src/dosbox.cpp

```diff
@@ -692,12 +692,13 @@ void DOSBOX_Init()
 
 	pint = secprop->Add_int("cycleup", always, 10);
 	pint->SetMinMax(1, 1000000);
-	pint->Set_help("Number of cycles added or subtracted with speed control hotkeys\n"
-	               "(10 by default).");
+	pint->Set_help("Number of cycles added with the speed up hotkey (10 by default).\n"
```

Comment:
Technically, it's called "increase cycles", so we should say "with the increase cycles" hotkey.

Reply:
Agreed